Source/BestProjectEver/Private/Core/GameModes/BPE_LobbyGameMode.cpp

```diff
@@ -31,3 +31,12 @@ void ABPE_LobbyGameMode::TravelToMatchMap()
 		}	
 	}
 }
+
+//----------------------------------------------------------------------------------------------------------------------
+void ABPE_LobbyGameMode::DestroyInactiveActor(AActor* ActorToDestroy)
+{
+	if(IsValid(ActorToDestroy))
+	{
+		ActorToDestroy->Destroy();
```

Comment:
This method is confusingly unnecessary. If you call this function on an inactive actor, or if you call this function on an active actor, or if you explicitly request an actor to be destroyed, there would be no difference.

Reply:
I corrected it in the commit <3